app/styles/patterns/global.css

```diff
@@ -65,7 +65,7 @@ html {
     width: 100%;
     /* Prevent elastic scrolling on the whole page */
     height: 100%;
-    font: 62.5%/1.65 "Open Sans", sans-serif;
+    font: 62.5%/1.65 -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif;
```

Comment:
just being a nerd reading your linked reference articles, should `"Open Sans"` be `"Droid Sans"`?

>font-family: -apple-system, BlinkMacSystemFont, 
    "Segoe UI", "Roboto", "Oxygen", 
    "Ubuntu", "Cantarell", "Fira Sans", 
    "Droid Sans", "Helvetica Neue", sans-serif;